nerdlets/container-explorer/container-heat-map.js

```diff
@@ -74,7 +74,7 @@ export default class ContainerHeatMap extends React.Component {
 
     return (
       <Heatmap
-        accountId={account.id}
+        accountIds={account.ids}
```

Comment:
Unless there's been a change to how props are passed into this component, then the `account` object most likely still just has a single `id` field. So you shouldn't need to update the naming (eg `ids`) and just keep it as `account.id`
```suggestion
        accountIds={[account.id]}
```